.github/workflows/_get_app_metadata.yml

```diff
@@ -0,0 +1,135 @@
+name: Output metadata on the application, inferred from the local 'ledger_app.toml' manifest or not
+
+on:
+  workflow_call:
+    inputs:
+      relative_app_directory:
+        description: |
+          The relative path in the repository where the application is built from (defaults to ".").
+
+          If the application is configured with a 'ledger_app.toml' manifest at its root, this
+          parameter is ignored.
+        required: false
+        default: .
+        type: string
+      compatible_devices:
+        description: |
+          The list of device(s) supported by the application.
+
+          Defaults to ["nanos", "nanox", "nanosp", "stax"]), except if the application is configured
+          with a 'ledger_app.toml' manifest at its root.
+        required: false
+        default: 'None'
+        type: string
+      pytest_directory:
+        description: |
+          The directory where the Python tests are stored (a `conftest.py` file is expected there).
```

Comment:
Same as with `relative_app_directory`, maybe even more because (for now?) this `pytest_directory` is optional in `ledger_app.toml` (some apps do not have tests).
So this parameter could still be relevant even if the manifest is present.